app.js

```diff
@@ -1,3 +1,7 @@
+if (parseInt(process.version.replace("v", "").split(".")[0]) < 14) {
```

Comment:
Try catch @NotExpectedYet maybe better in case process.version is not set?

Reply:
Yeah I'd say so, there's always the chance.